src/engine/ruleset/decoders/f5/f5-apm.yml

```diff
@@ -23,36 +23,34 @@ definitions:
+    # May 2016/05/08 07:27:59 labor low isqu: 01490167: :uis: Current snapshot ID: idolore updated inside session db for access profile: onse
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <~>:<~/ignore/:> <event.code>:<~/ignore/:><~content>
 
-    # Rule
-    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <event.provider>[<process.pid>]:<~/ignore/ ><~tmp.rsa.internal.messageid>:<~/ignore/ ><~tmp.rsa.misc.rule_name> \<\<<~tmp.rsa.internal.event_desc>>:<~/ignore/ >APM_EVENT=<event.action> | <user.name> | <~> ***<~tmp.rsa.misc.result>***
+    # iRules : https://clouddocs.f5.com/api/irules/
+    # October 2019/10/18 03:14:14 tem very-high giatnula[71]: Rule: enimadmi <<qui>: APM_EVENT=deny | aecon | sedq ***failure***
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <event.code>[<process.pid>]:<~/ignore/ ><~tmp.rsa.internal.messageid>:<~/ignore/ ><~tmp.rsa.misc.rule_name> \<\<<~tmp.rsa.internal.event_desc>>:<~/ignore/ >APM_EVENT=<event.action> | <user.name> | <~> ***<~tmp.rsa.misc.result>***
 
     # crond
     # March 2016/03/12 03:17:42 seq high crond[5738]: (ccaecat) veleumi
     # July 2016/07/18 18:40:50 lum high CROND[1675]: (sitvolup) CMD (cancel)
     # January 2019/01/19 13:25:23 volup very-high crond[4071]: (iconsequ) CMD (block)
     # April 2019/04/29 14:43:23 Nequepo high CROND[2977]: (emac) CMD (cancel)
-    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <event.provider>[<process.pid>]:<~/ignore/ >\(<~tmp.user_name>\) (?CMD \(<~tmp.user_command>\))<?~tmp.rsa.db.index>
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <event.code>[<process.pid>]:<~/ignore/ >\(<~tmp.user_name>\) (?CMD \(<~tmp.user_command>\))<?~tmp.rsa.db.index>
 
-    # syslog-ng - auditd - Rule - SMTP
+    # syslog-ng - auditd - SMTP
     # November 2019/11/01 10:16:48 erc low tasnu: [syslog-ng]
-    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <~tmp.rsa_misc_client>:<~/ignore/:> [<event.provider>]
-    # June 2018/06/04 20:44:15 lorumw high tdolo[3872]: syslog-ng:
-    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <~tmp.rsa_misc_client>[<process.pid>]:<~/ignore/ ><event.provider>:<~/ignore/ >
-    # April 2018/04/08 16:33:58 liq low mvolupta: syslog-ng:
-    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <~tmp.rsa_misc_client>:<~/ignore/:> <event.provider>:<~/ignore/ >
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <~tmp.rsa_misc_client>:<~/ignore/:> [<event.code>]
     # March 2017/03/18 18:24:33 ptasnula high syslog-ng[2638]: ill
-    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <event.provider>[<process.pid>]:<~/ignore/ ><~tmp.rsa_misc_client>
+    - event.original: <~> <@timestamp/%Y\/%m\/%d %T> <~> <log.level> <event.code>[<process.pid>]:<~/ignore/ ><~tmp.rsa_misc_client>
```

Comment:
What is it that is being ignored before the Timestamp?

Reply:
It's the month string representation